### ISSUE TITLE
Document ptyserv.py example

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5620]
|<img alt="lesleylam's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| lesleylam reported|
|-|-|
|Trac ID|trac#5620|
|Type|enhancement|
|Created|2012-04-06 20:47:31Z|

ptyserv.py in doc/core/examples needs to be documented.

Attachments:

* [ptyserv.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/832/8324766ce239d01947758ee7da7f8061beca0635/9f6d2740b4e12770c1dcf95328cd99754ea7de45.diff) (825 bytes) - added by lesleylam on 2012-04-06 21:07:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5620 5620
type__enhancement enhancement
reporter__lesleylam lesleylam
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1333745251000000 1333745251000000
changetime__1333819644000000 1333819644000000
version__None None
owner__thijs thijs
cc__jesstess cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to lesleylam|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="lesleylam's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|lesleylam commented|
|-|-|

Added documentation to ptyserv.py.  Diff file attached for review.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch. This is a trivial change so I'll use a .misc [ReviewProcess#Newsfiles news file] instead.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34130]) Apply ptyserv.diff: Explain how to run the ptyserver example.

Author: lesleylam
Reviewer: thijs
Fixes: [#5620](https://github.com/twisted/twisted/issues/5620)
